changelogs/unreleased/3851-update-classifiers-in-setup-py.yml

```diff
@@ -0,0 +1,6 @@
+change-type: patch
+issue-nr: 3851
+description: Update python versions in the setup.py classifiers
+destination-branches: [master]
```

Comment:
Shouldn't this go into iso5 as well?

Reply:
I wonder, is there a way for me to know towards which branches to push the changes ?

Reply:
I guess the documentation must mention somewhere that iso5 supports python 3.9 and iso4 python 3.6, but I could not point you to where that might be, so I can definitely not blame you for not knowing it :wink: